client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -1,5 +1,221 @@
+import { localize } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormButton from 'components/forms/form-button';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import ReduxFormRadio from 'components/redux-forms/redux-form-radio';
+import ReduxFormSelect from 'components/redux-forms/redux-form-select';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import ReduxFormToggle from 'components/redux-forms/redux-form-toggle';
+import SectionHeader from 'components/section-header';
+
+const JobSubmission = ( { isDisabled, submissionDuration, translate } ) => {
+	return (
+		<div>
+			<form>
+				<FormSection name="account">
+					<SectionHeader label={ translate( 'Account' ) }>
+						<FormButton compact
+							disabled={ isDisabled } />
+					</SectionHeader>
+					<Card>
+						<FormFieldset>
+							<ReduxFormToggle
+								disabled={ isDisabled }
+								name="isRequired"
+								text={ translate( 'Require an account to submit listings' ) } />
+							<FormSettingExplanation isIndented>
+								{ translate( 'Limits job listing submissions to registered, logged-in users.' ) }
+							</FormSettingExplanation>
+
+							<ReduxFormToggle
+								disabled={ isDisabled }
+								name="enableRegistration"
+								text={ translate( 'Enable account creation during submission' ) } />
+							<FormSettingExplanation isIndented>
+								{ translate( 'Includes account creation on the listing submission form, to allow ' +
+									'non-registered users to create an account and submit a job listing simultaneously.' ) }
+							</FormSettingExplanation>
+
+							<ReduxFormToggle
+								disabled={ isDisabled }
+								name="generateUsername"
+								text="Generate usernames from email addresses" />
+							<FormSettingExplanation isIndented>
+								{ translate( 'Automatically generates usernames for new accounts from the registrant\'s ' +
+									'email address. If this is not enabled, a "username" field will display instead.' ) }
+							</FormSettingExplanation>
+						</FormFieldset>
+
+						<FormFieldset>
+							<FormLabel>
+								{ translate( 'Role' ) }
+							</FormLabel>
+							<ReduxFormSelect
+								disabled={ isDisabled }
+								name="role">
+								<option value="editor">{ translate( 'Editor' ) }</option>
+								<option value="author">{ translate( 'Author' ) }</option>
+								<option value="contributor">{ translate( 'Contributor' ) }</option>
+								<option value="subscriber">{ translate( 'Subscriber' ) }</option>
+								<option value="teacher">{ translate( 'Teacher' ) }</option>
+								<option value="employer">{ translate( 'Employer' ) }</option>
+								<option value="customer">{ translate( 'Customer' ) }</option>
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Customer Name' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).